Source/WebCore/CMakeLists.txt

```diff
@@ -2147,6 +2147,12 @@ set(MODERN_MEDIA_CONTROLS_STYLE_SHEETS
     "${WEBCORE_DIR}/Modules/modern-media-controls/controls/watchos-media-controls.css"
 )
 
+if (PORT STREQUAL "GTK" OR PORT STREQUAL "WPE")
```

Comment:
We would normally duplicate this code between PlatformGTK.cmake and PlatformWPE.cmake, to keep it out of the cross-platform CMakeLists.txt.

Reply:
Or put it in `PlatformGLib.cmake`, but in this case we need to have the final value of `MODERN_MEDIA_CONTROLS_STYLESHEET` ready before the `add_custom_command()` that defines the target for generating `ModernMediaControls.css` right below... but `PlatformXXX.cmake` gets included by the build system _after_ the `CMakeLists.txt` file was processes, so I'm afraid we don't have a better way in this case.